src/Razor/src/Microsoft.VisualStudio.Editor.Razor/Telemetry/TelemetryReporter.cs

```diff
@@ -55,7 +55,6 @@ public void ReportFault(Exception exception, string? message, params object?[] @
         {
             if (exception is OperationCanceledException { InnerException: { } oceInnerException })
```

Comment:
`oceInnerException` isn't used any more, so can be removed, plus this has the benefit of not reporting telemetry for operation canceled exceptions even if they don't have an inner exception.

```suggestion
            if (exception is OperationCanceledException)
```

Reply:
Should we report the inner exception if it's not oce?

Reply:
Yeah, I guess that was the original intention of this code, so probably a good idea.